saleor/account/migrations/0041_permissions_to_groups.py

```diff
@@ -39,11 +39,8 @@ def get_counter_value(Group):
 def create_permissions_mapping(User):
     """Create mapping permissions to users and potential new group name."""
     mapping = defaultdict(set)
-    users = (
-        User.objects.filter(user_permissions__isnull=False)
-        .distinct()
-        .prefetch_related("user_permissions")
-    )
+    users = User.objects.filter(user_permissions__isnull=False).distinct().iterator()
```

Comment:
Why we use `distinct()` here?

Reply:
Because it returns duplicated items when a user has more than one `user_permissions`

Reply:
👮‍♀️